src/test/java/org/opensearch/security/IntegrationTests.java

```diff
@@ -1045,9 +1046,8 @@ public void testSecurityIndexSecurity() throws Exception {
             encodeBasicHeader("nagilum", "nagilum")
         );
         Assert.assertEquals(HttpStatus.SC_FORBIDDEN, res.getStatusCode());
-        // res = rh.executePostRequest(".opendistro_security/_freeze", "",
-        // encodeBasicHeader("nagilum", "nagilum"));
-        // Assert.assertTrue(res.getStatusCode() >= 400);
+        res = rh.executePostRequest(".opendistro_security/_freeze", "", encodeBasicHeader("nagilum", "nagilum"));
+        Assert.assertTrue(res.getStatusCode() >= 400);
```

Comment:
nit: Is there a specific status code we can get? such as 403?

Reply:
not sure why this was added as >=400 originally,  I will correct this to the expected return code which is 400